### PR TITLE
Replacing effective_created with new function and renaming to effecti…

### LINKED ISSUE #314: Federate out edited timestamp when editing content
Currently when federating content edits, the created timestamp is sent out. Send instead the edited timestamp.

This makes edit federate more reliably with for example Hubzilla.

### PR DESCRIPTION
**Pull request by ghost**
This fixed #314 in the way discussed in the thread

### CODE REVIEW

File: socialhome/content/models.py [hunk 1]
```diff
@@ -260,10 +260,10 @@ def is_nsfw(self):
         return self.text.lower().find("#nsfw") > -1
 
     @property
-    def effective_created(self):
-        if self.remote_created:
+    def effective_modified(self):
+            if self.remote_created:
```
Comment (jaywink) [inline socialhome/content/models.py]: Indentation mistake here.

**Review by jaywink — CHANGES_REQUESTED** [review]
Looks good except for the small indentation mistake.

Btw, please if possible update the existing PR's when pushing changes. Opening a new PR creates unnecessary noise and also makes it impossible to keep track of previous review comments. This is why git has amending commits and force pushing.

Thanks! 👍

**Comment by ghost** [comment]
Fixed. Sorry about that.

**Comment by jaywink** [comment]
A test is failing because you didn't replace all occurrences of `effective_created`.

**Comment by ghost** [comment]
That's very odd. I ran the test on my image before committing it... I'll need to look into this later

**Comment by ghost** [comment]
Can I see where it failed? My local tests run fine and my branch has no unpushed commits.

**Comment by jaywink** [comment]
Yes, Github pr's have a link to the Travis test run, see the "Details" link. It looks to me like the error is because you did not replace all occurrences of `effective_created` when removing it. It's good to do a full code search when removing properties and methods.

**Review by jaywink — APPROVED** [review]

**Comment by jaywink** [comment]
Thanks! 🎉

**Pull request merged**